src/middlewared/middlewared/plugins/update_/install_freebsd.py

```diff
@@ -73,6 +73,15 @@ def install_manual_impl(self, job, path, dest_extracted):
             raise CallError('Uploaded file is not a manual update file')
 
     def _install_scale(self, job, path):
+        if self.middleware.call_sync('system.product_type') == 'ENTERPRISE':
+            minio = self.middleware.call_sync('service.query', [('service', '=', 's3')], {'get': True})
+            if minio['enable'] or minio['state'] == 'RUNNING':
+                raise CallError(
+                    "There are active configured services on this system that are not present in the new version. To "
+                    "avoid any loss of system services, please contact iXsystems Support to schedule a guided upgrade. "
+                    "Additional details are available from https://www.truenas.com/docs/scale/scaledeprecatedfeatures/."
```

Comment:
I spoke to the docs team and this is the URL we need to use for CORE
```suggestion
                    "Additional details are available from https://www.truenas.com/docs/core/13.0/gettingstarted/deprecations."
```